webapps-schema/src/main/resources/schema/opensearch/create/index/identity-groups.json

```diff
@@ -5,7 +5,7 @@
       "id": {
         "type": "keyword"
       },
-      "groupKey": {
+      "key": {
```

Comment:
Do you know how to repair the local state after a change like this? I've struggled getting primary and secondary storage in sync when I've made similar changes.

Reply:
Do you mean for local testing?

Reply:
Yes, for my local dev env.

Reply:
I assume the issue comes from IT tests, where ES is used. Have you tried clearing the Docker volumes, maybe even some stale test containers that might have been built with older Zeebe/Exporter snapshots?

Reply:
The issue is with the test data I have in my local env already: I've created users, groups, tenants and roles via the API or even the UI and I'd like to keep as much of that as possible.

Reply:
I see. I usually don't keep local test data, but I understand how it is useful when developing the frontend. AFAIK, the (Zeebe) ES indexes were versioned, so migration with the former exporter/importer setup wasn't an issue between different Camunda 8 versions. However, in your local setup, you're probably always using a `SNAPSHOT` version, and that creates conflicts.

I would suggest reaching out to the Operate or Tasklist teams, they may have more experience with this type of local setup.